### DISCUSSION TITLE
Install Error: I'm trying to compile but it stops at iped-viewers-impl

### DISCUSSION THREAD

**Discussion by gustavojm27** [sepinf-inc/IPED#192] [Q&A] [answered]
Msg error:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project iped-viewers-impl: Compilation failure: Compilation failure:
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[21,26] package javafx.application does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[22,26] package javafx.beans.value does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[23,26] package javafx.beans.value does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[24,25] package javafx.concurrent does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[25,26] package javafx.embed.swing does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[26,20] package javafx.scene does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[27,27] package javafx.scene.layout does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[28,24] package javafx.scene.web does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[29,24] package javafx.scene.web does not exist
[ERROR] /C:/IPED/iped-viewers/iped-viewers-impl/src/main/java/dpf/sp/gpinf/indexer/ui/fileViewer/frames/HtmlViewer.java:[40,13] cannot find symbol

**Comment by lfcnassif** [comment, marked as answer]
Please read the build instructions on Readme.md file. You need javaFx to compile from source. You can use oracle jdk 8 or liberica jdk 8 FULL
